src/clients/HubPoolClient.ts

```diff
@@ -12,11 +12,13 @@ export class HubPoolClient extends clients.HubPoolClient {
     configStoreClient: clients.AcrossConfigStoreClient,
     deploymentBlock?: number,
     chainId = 1,
-    eventSearchConfig: MakeOptional<EventSearchConfig, "toBlock"> = { fromBlock: 0, maxBlockLookBack: 0 }
+    eventSearchConfig: MakeOptional<EventSearchConfig, "toBlock"> = { fromBlock: 0, maxBlockLookBack: 0 },
+    timeToCache = 60 * 60
```

Comment:
```suggestion
    timeToCache?: number
```

Why not just let this be undefined and use the default from the superclass?